.github/workflows/format.yml

```diff
@@ -13,16 +13,18 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v2
       - name: Run format
-        run: dartfmt --dry-run --set-exit-if-changed .
+        run: dart format --set-exit-if-changed .
 
   import_sorter:
     runs-on: ubuntu-latest
-    container:
-      image: google/dart:latest
     steps:
       - name: Checkout Repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
+      - name: Setup Dart
+        uses: dart-lang/setup-dart@v1
+        with:
+          sdk: stable
```

Comment:
for consistency, could you apply the same changes to the `dartfmt` job above?
- remove `container`
- add `Setup Dart` step
- rename job to `format`

Reply:
Have made the requested changes